guides/common/modules/ref_supported-clients.adoc

```diff
@@ -0,0 +1,61 @@
+[id="supported-clients_{context}"]
+= Supported Clients
+
+{Project} supports the following operating systems and architectures.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifdef::orcharhino[]
+* Alma Linux
+* Amazon Linux
+endif::[]
+ifndef::satellite[]
+* CentOS
+endif::[]
+ifndef::orcharhino,satellite[]
+* CoreOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* Junos
```

Comment:
I'm not sure Junos still works. It's a bit of an odd one since it's a router OS. I wouldn't list it.